plugins/Login/PasswordResetter.php

```diff
@@ -179,8 +191,8 @@ public function initiatePasswordResetProcess($loginOrEmail, $newPassword)
         } catch (Exception $ex) {
             // remove password reset info
             $this->removePasswordResetInfo($login);
-
-            throw new Exception($ex->getMessage() . Piwik::translate('Login_ContactAdmin'));
+            $this->logger->error($ex); // log exception details to log
```

Comment:
@geekdenz this would be potentially logged to the user screen. We don't have a functionality to only log to logs and in this case would likely also not want to log potential information to logs.

Could above information also be triggered for different errors where we maybe still want to show the original error message?

Reply:
We could catch the specific Exception (find out which one - AuthenticationException or similar) and otherwise also log the message. But in case of an AuthenticationException, log:

> An unspecified error occured, please contact an admin.

or similar. Would this be desirable?

Reply:
we don't really need to log anything in particular here as the message will be shown to the user anyway.